ci-operator/step-registry/ipi/conf/vsphere/dns/ipi-conf-vsphere-dns-commands.sh

```diff
@@ -90,3 +89,14 @@ cat > "${SHARED_DIR}"/dns-delete.json <<EOF
 }]}
 EOF
 
+aws route53 change-resource-record-sets --hosted-zone-id "$hosted_zone_id" --change-batch file:///"${SHARED_DIR}"/dns-create.json > result.json
+
+cat result.json
+
+id=$(jq -r '.ChangeInfo.Id' < result.json)
```

Comment:
I think you can use aws cli built in --query https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-output.html#cli-usage-output-filter

```suggestion
aws route53 change-resource-record-sets --hosted-zone-id "$hosted_zone_id" --change-batch file:///"${SHARED_DIR}"/dns-create.json --query 'ChangeInfo.Id' --output text

cat result.json

id=$(jq -r '.ChangeInfo.Id' < result.json)
```